### DISCUSSION TITLE
Medics won't heal a husk infection until it's too late

### DISCUSSION THREAD

**Discussion by DownloadableContent5021** [FakeFishGames/Barotrauma#14726] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you have a crewmember tasked with "heal and rescue", they won't heal a husk infection because they always consider it to be a "minor injury", so they "won't waste any meds on it". It's not until the husk infection hits 100% that the AI will try to heal it, and at this point, calyxanide won't heal the affliction anymore; it'll just kill the husk.
<img width="1680" alt="Screenshot 2024-09-26 at 12 42 44 PM" src="https://github.com/user-attachments/assets/1ec30ede-d0bc-40c3-a975-a02e4cc9cf8f">

### Reproduction steps

1. Task a crewmate with "heal and rescue"
2. Infect yourself with a husk infection
3. Watch as the medic will only try to heal you once the husk infection hits 100%

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.9.0 (unstable)

### Which operating system did you encounter this bug on?

MacOS

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14504
